changelogs/fragments/7873-lxd-instance.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - lxd connection plugin - refer to containers/VMs as instances (https://github.com/ansible-collections/community.general/pull/7873).
+  - lxd inventory plugin - refer to containers/VMs as instances (https://github.com/ansible-collections/community.general/pull/7873).
```

Comment:
Docs-only changes aren't listed in the changelog, so please remove this file. Thanks :)